src/jit/emitxarch.cpp

```diff
@@ -2920,174 +2920,283 @@ void emitter::emitInsStoreLcl(instruction ins, emitAttr attr, GenTreeLclVarCommo
 
 regNumber emitter::emitInsBinary(instruction ins, emitAttr attr, GenTree* dst, GenTree* src)
```

Comment:
Since you've got this function in your head now, it would be useful to write a good (standard format) function header comment describing it.

Reply:
Added a function header.